src/DotNetWorker/Context/Features/ConverterBidingFeature.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    internal class ConverterBidingFeature
```

Comment:
Spelling?

Reply:
Should have left this type out.... was mostly stubbing this. I'll actually delete this class from this change